gradle.properties

```diff
@@ -17,7 +17,7 @@ dropwizardMetricsVersion=3.1.2
 eventuateUtilVersion=0.2.0.RELEASE
 eventuateCommonVersion=0.8.0.RELEASE
 
-eventuateMessagingKafkaVersion=0.4.0.RELEASE
+eventuateMessagingKafkaVersion=0.1.0-SNAPSHOT
 eventuateMessagingActiveMQVersion=0.4.0.RELEASE
```

Comment:
These need to be upgraded to the latest released versions of the libraries (i.e. yesterday's releases)

Reply:
will do